.github/workflows/ci.yml

```diff
@@ -23,7 +23,15 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [3.6.4, 3.5.2, 3.4.3, 3.3.4, 3.2.2, 2.13.16, 2.12.20]
+        scala:
```

Comment:
Just a note: The reformatting here was done by the githubWorkflowGenerate sbt task, not me